### DISCUSSION TITLE
License change from Apache 2.0 to MIT?

### DISCUSSION THREAD

**Discussion by arianman** [eclipse-edc/.github#34] [Q&A] [answered]
Hi all,

If we fork a repo (for example for the extensions) to work on it on an organizational GitHub, could we change the license from Apache 2.0 to MIT? considering the fact that we would want to contribute back to the EDC at some point! 

In general I know these two license could go together, but would be interested to know how it should work from your perspective. (The need for change comes from the company policies, and is not a personal preference). 

Many thanks!

**Comment by mspiekermann** [comment, marked as answer]
Hi @arianman, the EDC has Apache 2 license and so would have all the files you fork into your organizational repo. You can choose the license for your **additional** code yourself and e.g. use MIT. For extensions to be listed on the [known-friends list](https://github.com/eclipse-edc/Connector/blob/main/known_friends.md) that would be totally fine and a model how you can work with the EDC. However, contributions that directly contribute to the EDC project must be submitted under the Apache 2 license and are subject to Eclipse Foundations rules and review. 

> In general I know these two license could go together

We are not aiming on a dual license for the EDC, especially as both, Apache 2 and MIT, have comparable scopes, which would narrow the benefits. So if you decide to contribute your code to the EDC, which we would very much appreciate, it might be conceivable to do so under Apache 2 as well.

**Reply by arianman** [reply]
Hi @mspiekermann, thanks you for the explanation. it helped a lot!
